packages/terra-list/src/List.jsx

```diff
@@ -98,9 +109,21 @@ const List = ({
   refCallback,
   role,
   ariaSelectionStyle,
+  isDraggable,
+  onDragEnd,
   ...customProps
 }) => {
-  const theme = React.useContext(ThemeContext);
+  const theme = useContext(ThemeContext);
+  const [listItem, setlistItem] = useState([]);
+
+  useEffect(() => {
+    if (!(Array.isArray(children))) {
+      if (children) setlistItem([children]);
+    } else {
+      setlistItem(children);
+    }
```

Comment:
```suggestion
    if (Array.isArray(children)) {
      setlistItem(children);
    } else if (children) {
      setlistItem([children]);
    }
```
to prevent setState on undefined values

Reply:
Changes done Thanks